concent_api/conductor/views.py

```diff
@@ -24,6 +24,7 @@
 @provides_concent_feature('conductor-urls')
 @require_POST
 @csrf_exempt
+@transaction.non_atomic_requests(using='storage')
 @non_nesting_atomic(using='storage')
 def report_upload(_request: HttpRequest, file_path: str) -> HttpResponse:
```

Comment:
You're turning atomic requests off and then back on immediately:
``` python
@transaction.non_atomic_requests(using='storage')
@non_nesting_atomic(using='storage')
```

This can only detect something if someone calls our view from within a transaction. But we never call views explicitly. We let Django router do this. So in practice it's not necessary.

It seems redundant but it's not wrong. But if you prefer it that way, you should do the same with `send/`, `receive/` and any other views for consistency.

Reply:
> But we never call views explicitly. We let Django router do this. So in practice it's not necessary.

I agree, but I would prefer to keep it for consistency, in case if it would be used differently and someone would miss this. And views in `send/` and `receive` are also using `@transaction.non_atomic_requests()` already.